package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "react-native-billing-sdk",
-    "version": "0.3.2",
```

Comment:
@AbdullahKayahan burayı geri alabilir misin, release-it ile ben burayı halledeceğim